docs/source/cluster_management/sweep/standard-sweep.rst

```diff
@@ -36,6 +36,7 @@ Note that some of these parameters are just used as a hint. Sweep dynamically mo
    :widths: 20, 20, 40, 200
 
    ``enabled``, "Only specified in config", "true", "Whether the background sweeper should run."
+   ``sweepThreads``, "Only specified in config", "1", "The number of threads to run sweep with. Changes require a restart of any service node to take effect. Not recommended for Cassandra KVS."
```

Comment:
Might be worth specifying here the contention for the backup lock. Outside of that, is there a reason to avoid this?

Reply:
Sweep is still quite a heavy operation for Cassandra, and extra sweep threads sound like a recipe for trouble, and we don't need to take that risk given the proximity of targeted sweep. I'll add something about the backup lock, though.